otterdog/eclipse-velocitas.jsonnet

```diff
@@ -146,6 +165,10 @@ orgs.newOrg('eclipse-velocitas') {
     },
     orgs.newRepo('devenv-runtime-k3d') {
       allow_update_branch: false,
+      archived: false,
```

Comment:
@BjoernAtBosch should we archive this now?

Reply:
Motivation:
1. Not usage detected
2. Archiving only made repo "read-only". Currently, no maintain is expected. All functionality moved into devenv-runtimes repo

Reply:
If it's as easy as setting those flags to true, we should (at least for the deprecated runtime repos).

Reply:
yes runtime-k3d is not used anymore and should be archived so please add this flag